CONTRIBUTING.md

```diff
@@ -38,7 +38,7 @@ Explain the problem and include additional details to help maintainers reproduce
 * **Use a clear and descriptive title** for the issue to identify the problem.
 * **Describe the exact steps which reproduce the problem** in as many details as possible.
 * **Provide your pyproject.toml file** in a [Gist](https://gist.github.com) after removing potential private information (like private package repositories).
-* **Provide specific examples to demonstrate the steps to reproduce the issue**. Include links to files or GitHub projects, or copy-paste-able snippets, which you use in those examples.
+* **Provide specific examples to demonstrate the steps to reproduce the issue**. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples.
```

Comment:
The slash makes it seem like "copy or paste", and "pasteable" is awkward. I suggest:
> Include links to files or GitHub projects, or snippets that are easy to copy and paste, which you use in those examples.

Reply:
I'll agree that "pasteable" is awkward, but "copy/paste" is fairly common parlance. In any case, this isn't something I changed myself, I just aligned it to the other `CONTRIBUTING.md`.

Given that there's merge conflicts and these two `CONTRIBUTING.md` have gotten even more out-of-sync, at this point I think it would be better for the app root `CONTRIBUTING.md` to just link to `docs/docs/contributing.md`

Also, obligatory: ["Good developers copy; great developers paste."](https://twitter.com/kelseyhightower/status/1022837097743319040?s=20)